discord-impl/src/main/java/net/starpye/quiz/discordimpl/command/HelpCommand.java

```diff
@@ -0,0 +1,45 @@
+package net.starpye.quiz.discordimpl.command;
+
+import discord4j.core.spec.MessageCreateSpec;
+import discord4j.rest.util.Color;
+
+import java.awt.*;
+import java.util.Collection;
+
+public class HelpCommand implements QuizCommand {
+
+    private Collection<? extends QuizCommand> commands;
+
+    public HelpCommand(Collection<? extends QuizCommand> commands) {
+        this.commands = commands;
+    }

```

Comment:
Passing directly the `QuizCommand` enable the class to access not only the description and the name (which is required) but also to execute the command. We could add (yeet) another interface to better encapsulate the behavior.
This is not required, though so do as you wish